index.js

```diff
@@ -173,13 +274,13 @@ module.exports.toASCII = (domainName, useSTD3, processingOption, verifyDnsLength
   });
 
   if (verifyDnsLength) {
-    const total = labels.slice(0, labels.length - 1).join(".").length;
-    if (total.length > 253 || total.length === 0) {
```

Comment:
Oh, I misunderstood your question. This one had me confused as well. The "root label" the spec is referring to is the implied empty label that all domain names have, but is generally omitted with its dot. However it is present in output from tools like `dig` (note the trailing dot after `github.com`):

```
;; QUESTION SECTION:
;github.com.                    IN      A

;; ANSWER SECTION:
github.com.             30      IN      A       192.30.255.113
github.com.             30      IN      A       192.30.255.112
```

Since it is usually omitted, we don't have to do anything extra to account for that.